### ISSUE TITLE
TinyMCE does not restore alt text and captions on figures

### ISSUE THREAD

**Issue by iOvergaard** [umbraco/Umbraco-CMS#17148]
### Which Umbraco version are you using? (Please write the *exact* version, example: 10.1.0)

14.2.0

### Bug summary

When you pick a media item and put in alt text and caption, and you then edit the inserted image, the fields are reverted to their defaults.

### Specifics

The TinyMCE property editor.

### Steps to reproduce

1. Insert a media item and give it a caption
2. Verify it inserts a `<figure>` with a `<figcaption>` in the source
3. Edit the image again, the fields are now blank / reset to default

### Expected result / actual result

The caption field should be filled out with the stored caption. The alt text field, which is retrieved from the media item itself, should continue doing so unless the user has overridden the value.

**Comment by iOvergaard** [comment]
This is fixed by https://github.com/umbraco/Umbraco.CMS.Backoffice/pull/2347 and will be released in 15.0.
